modules/ztp-sno-node-reboot-scenarios.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-reference-cluster-configuration-for-vdu.adoc
+
+:_content-type: CONCEPT
+[id="ztp-sno-node-reboot-scenarios_{context}"]
+= SNO node reboot scenario
+
+On a {sno-caps} cluster and in {product-title} clusters generally a situation might arise due to a node reboot or kubelet restart where an associated device plugin restart has the affect of causing the application pod to fail with the `UnexpectedAdmissionError` error. This error is reported due to the fact that application pods that need devices come up before the pod that is serving the devices because there is no way to control the pod restart order.
+
+While this behavior is expected, it can lead to a pod remaining on the cluster  even when it has failed to deploy successfully and it will continue to report `UnexpectedAdmissionError`. This issue might cause undue concern particularly in a {sno-caps} deployment, where the option to drain the node is unavailable.
+In multi-node deployments, you drain a node, which results in the transfer of workloads to other worker nodes in the deployment.
```

Comment:
Can we add a link to documentation with details on how to drain the nodes here. Maybe [this](https://docs.openshift.com/container-platform/4.13/nodes/nodes/nodes-nodes-working.html) one?  I am guessing we'd have to update the version depending on the targeted version?